llvm/test/CodeGen/ARM/misched-branch-targets.mir

```diff
@@ -0,0 +1,166 @@
+# RUN: llc -o - -run-pass=machine-scheduler -misched=shuffle %s | FileCheck %s
```

Comment:
Please generate the file using `update_mir_test_checks.py`.

Reply:
The script generates many more check lines than necessary, so I prefer to keep the testcase handmade.

Reply:
In other patches that I authored, reviewers strongly suggested that I use the automatic test update scripts, even if it generates more CHECK lines than necessary. In this case, how do I know that the output doesn't diverge in the two llc invocations, within lines you didn't check? If you used `update_mir_test_checks.py`, it would automatically bail out on diverging outputs.

Reply:
Each individual test case is expected to test for specific properties of the compiler output. A test case is not a detector for irrelevant changes. The minimum required amount of check lines means the test won't need to be updated for unrelated changes and in the same time makes is more clear what the test, well, tests.